src/app/public/src/modules/nav/nav.service.ts

```diff
@@ -120,7 +129,37 @@ export class StacheNavService {
       .join(' ');
   }
 
-  private clone(thing: any): any {
-    return JSON.parse(JSON.stringify(thing));
+  private sortRoutes(routes: StacheNavLink[]): StacheNavLink[] {
+
+    const sortByName = (a: any, b: any): number  => {
+      if (a.name.toLowerCase() < b.name.toLowerCase()) {
+        return -1;
+      } else if (a.name.toLowerCase() > b.name.toLowerCase()) {
```

Comment:
`else if` is redundant because you're using `return`.
```
if (a.name.toLowerCase() < b.name.toLowerCase()) {
  return -1;
}

if (a.name.toLowerCase() > b.name.toLowerCase()) {
  return 1;
}
```